deploy.sh

```diff
@@ -3,4 +3,42 @@
 if [ "${TRAVIS_BRANCH}" == "master" ] && [ "${TRAVIS_PULL_REQUEST}" == "false" ]; then
     docker login -u "${QUAY_IO_USERNAME}" -p "${QUAY_IO_PASSWORD}" quay.io 
     make push-image-rbdplugin push-image-cephfsplugin
+
+    set -e
+
+    mkdir -p tmp
+    pushd tmp > /dev/null
+
+    curl https://raw.githubusercontent.com/helm/helm/master/scripts/get > get_helm.sh
+    chmod 700 get_helm.sh
+    ./get_helm.sh
+
+    git clone https://github.com/ceph/csi-ceph
+    git clone https://github.com/ceph/ceph-csi
+
+    mkdir -p csi-ceph/docs
```

Comment:
how about one chart per directory? `docs/rbd` for rbd

Reply:
I don't think it will index correctly then?